### PR TITLE
HPCC4J-487 Add support for WsCloud

### PR DESCRIPTION
**Pull request by rpastrana**
- Adds WsCloud WSDL
- Adds Axis2 generated stub and corresponding mvn profile
- Adds corresponding junit

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [x] I have created a corresponding JIRA ticket for this submission
- [x] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [x] The change has been fully tested:
  - [ ] This change does not cause any existing JUnits to fail.
  - [ ] I have include JUnit coverage to test this change
  - [ ] I have performed system test and covered possible regressions and side effects.
- [ ] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] This change fixes the problem, not just the symptom

### CODE REVIEW

**Comment by rpastrana** [comment]
@jpmcmu please review

File: wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsCloudClient.java [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+        }
+    }
+
+    /**
+     * <p>getPods.</p>
+     *
+     * @return a JSON formatted list of available pods - in raw k8s format.
+     * @throws java.lang.Exception if any.
+     */
+    public String getPods() throws Exception
```
Comment (jpmcmu) [inline wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsCloudClient.java:233]: Returning raw k8s format here is interesting; I don't think we should take on providing consistency here, but it does lead me to wonder what the use cases for this are?

**Review by jpmcmu — APPROVED** [review]
@rpastrana Looks good, question about the raw k8s format, but no real issues.

File: wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsCloudClient.java (hunk 1 above)
Comment (rpastrana) [reply wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsCloudClient.java:233]: Agreed, I'm a bit surprised the ESP service was designed this way as well, typically they provide well defined response structures.
The use case is likely related to the new ECLWatch interface which lists hpcc cluster pods, and also uses those pod names as potential filters in the new log viewer.
As far as wsclient, this is added for completeness.

**Pull request merged**